tests/src/test_helpers/ssl_helpers.c

```diff
@@ -1638,6 +1638,10 @@ int mbedtls_test_ssl_tls13_populate_session(mbedtls_ssl_session *session,
     session->resumption_key_len = 32;
     memset(session->resumption_key, 0x99, sizeof(session->resumption_key));
 
+#if defined(MBEDTLS_SSL_EARLY_DATA)
+    session->max_early_data_size = 0x87654321;
```

Comment:
Where does the value 0x87654321 come from? is it only one test value?

Reply:
It is test value. This is for checking if the data is save/load correctly. So I pickup this value.